vue-frontend/src/pages/mobile-app-development.vue

```diff
@@ -41,13 +47,6 @@ export default {
   setup() {
     var seoData = config.MOBILE_APP_DEVELOPMENT_SEO_META_DATA;
     useMeta({
-      meta: [
```

Comment:
This should be removes when page is ready to deploy on production not in develoment